### DISCUSSION TITLE
MSIN%40CMU/

### DISCUSSION THREAD

**Discussion by giscus** [opencsapp/opencsapp.github.io#314] [Announcements]
# MSIN%40CMU/

None

https://opencs.app/MSIN%40CMU/

**Comment by guapisolo** [comment]
MSIN@CMU 24fall 利好竞赛和实习选手。提供一个 dp：985 cs，91/100 3.9/4.0 rk3%，T107无g，3icpc金1ccpc金，一段大厂实习。只有四个小文书，没有sop和ps。1.竞赛/顶会/其他获奖 2.专业经历(实习or全职) 3.团队合作 4.代码能力
